docs/source/userdoc/requirements/interns/adna-njuhovic/GPIO_requirements/informational_requirements.rst

```diff
@@ -0,0 +1,247 @@
+===============================
+Informational Requirements
+===============================
+
+.. req:: label
+   :id: REQ_INT3_1732817300
+   :status: Draft
+   :date-released:
+   :priority: Low
+   :submitted-by: Adna Njuhovic
+   :modified-by:
+   :category: Functional
+   :safety-asil: 
+   :references: 
+   :verification-and-validation: 
+
+   GPIO Pin Configuration
+   =======================
+   The GPIO pin configuration system shall support mode selection for each pin, ensuring the ability to configure each GPIO pin to one of four distinct modes: input, output, analog, and alternate function.
```

Comment:
"The GPIO pin configuration system shall support mode selection for each pin" ovaj dio je funkcionalni rq, ali posto si vec spominjala ovo u funkcionalnim rqs, mozes obrisati ovo i ostavi samo da svaki pin ima 4 moguca moda